### ISSUE TITLE
[Bug] Crashes when opening window

### ISSUE THREAD

**Issue by rowbawts** [flathub/nz.mega.MEGAsync#18]
When I click the tray icon to see the window it shows for a couple seconds then closes, I tried running it from the Terminal to see if there was an error and there isn't it just closes.

it seems to have started with the recent update as I was using it before without issue.

**Comment by tim77** [comment]
Can you please provide more details? What distro, what DE? And terminal output. App and recent update works fine for me.

**Comment by rowbawts** [comment]
Fedora 35 Silverblue - GNOME 41.3, there is no terminal output explaining this issue as I tried to check for it, does the app have a log?

**Comment by rowbawts** [comment]
It seems to me like it might be an upstream issue as I checked their GitHub repository as the sync client is open source and hosted on here and I saw some open issues talking about segfaults so since there isn't any error being produced in any log or the terminal it seems likely a segfault is the culprit

> Same in Fedora 34 (workstation) and Gnome version 40.7. It works in the background, but opening the window or settings crashes the app. No specific logs can be found about what's happening.

**Comment by Barraguesh** [comment]
@rowbawts installed the native rpm version and 0 problems so far. Windows open and it doesn't crash at all. Same system where it crashed in this Flatpak version.

**Comment by rowbawts** [comment]
> @rowbawts installed the native rpm version and 0 problems so far. Windows open and it doesn't crash at all. Same system where it crashed in this Flatpak version.

Well that's good to narrow it down to a Flatpak-specific issue, thanks for doing that I thought of trying it but I'm running Silverblue and it takes a few extra steps than a normal rpm install to get it on my system.

**Comment by tim77** [comment]
> there is no terminal output explaining this issue as I tried to check for it, does the app have a log?

@rowbawts could you please also try to run app with:
```
flatpak run -vv nz.mega.MEGAsync
```
and post output here.

**Comment by rowbawts** [comment]
There's a ton of output so I'm attaching it as a file to not clutter the issue page.
[output.txt](https://github.com/flathub/nz.mega.MEGAsync/files/8102356/output.txt)

**Comment by aunetx** [comment]
It crashes for me too, tell me if I can do anything

**Comment by JulianGro** [comment]
Same issue here on Debian Bookworm with Cinnamon.

**Comment by JulianGro** [comment]
Output from `flatpak run -vv nz.mega.MEGAsync`: https://paste.ubuntu.com/p/VmTG4tzFMb/

**Comment by edwinakatosh** [comment]
Reverting to an older commit worked for me:

```
flatpak remote-info --log flathub nz.mega.MEGAsync
flatpak update --commit=daef3245111ab602256daae1000cbc05bbe341f01484e823af2e834160a218ba nz.mega.MEGAsync
```

**Comment by rowbawts** [comment]
> Output from `flatpak run -vv nz.mega.MEGAsync`: https://paste.ubuntu.com/p/VmTG4tzFMb/

`Gtk-Message: 21:20:25.305: Failed to load module "xapp-gtk3-module"`

That's probably the most suspicious line to indicate the issue to my knowledge.

**Comment by JulianGro** [comment]
> > Output from `flatpak run -vv nz.mega.MEGAsync`: https://paste.ubuntu.com/p/VmTG4tzFMb/
> 
> `Gtk-Message: 21:20:25.305: Failed to load module "xapp-gtk3-module"`
> 
> That's probably the most suspicious line to indicate the issue to my knowledge.

As far as I know that isn't a problem actually. It supposedly comes from Cinnamon specific functionality. Your output doesn't contain that for example.

**Comment by JulianGro** [comment]
> Reverting to an older commit worked for me:
> 
> ```
> flatpak remote-info --log flathub nz.mega.MEGAsync
> flatpak update --commit=daef3245111ab602256daae1000cbc05bbe341f01484e823af2e834160a218ba nz.mega.MEGAsync
> ```

I can confirm that commit daef3245111ab602256daae1000cbc05bbe341f01484e823af2e834160a218ba works.
I tried every newer commit as well and none of them worked.

**Comment by rowbawts** [comment]
Yeah you're right I guess we should report this to the upstream issue tracker then.

**Comment by SayanChakroborty** [comment]
Any progress on this?

**Comment by loafylemon** [comment]
```
OS: Pop!_OS 22.04 LTS x86_64 
Kernel: 6.0.6-76060006-generic 
DE: GNOME 42.3.1
Flatpak: 1.12.7
MEGAsync: 4.7.2.0
```

As soon as I click "Show status" or go to settings and select the "Accounts" tab, the application crashes. 

[logs.txt](https://github.com/flathub/nz.mega.MEGAsync/files/10051133/logs.txt)

Possible related Flatpak issue (?):

https://github.com/flatpak/flatpak/issues/5047

**Comment by Lokawn** [comment]
I encountered similar issue, Megasync automatically closing after sometime of launching.
My /etc/env contains `QT_QPA_PLATFORM=wayland`;
setting environment for Megasync using flatseal to `QT_QPA_PLATFORM=xcb` solved the issue.
Setting `QT_QPA_PLATFORM="wayland;xcb"` in env also didn't work.

**Comment by nicolasdb** [comment]
same here on 
```
Operating System: Manjaro Linux
KDE Plasma Version: 5.26.5
KDE Frameworks Version: 5.103.0
Qt Version: 5.15.8
Kernel Version: 6.1.12-1-MANJARO (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i7-3520M CPU @ 2.90GHz
Memory: 11,5 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 4000
Manufacturer: LENOVO
Product Name: 2344CTO
System Version: ThinkPad T430
```

I did a fresh install of manjaro on the 13 fev. 23 , then install mega with flatpack, it worked for a time, then I ran on this problem after an update. I'm not sure if it come from an update system or an update flatpak.
It's still sync files ok but crash as soon as I click the icon.

**Comment by loafylemon** [comment]
New hardware, same problem.

> Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
Qt: Session management error: None of the authentication protocols specified are supported
amdgpu: unknown (family_id, chip_external_rev): (145, 1)
libGL error: failed to create dri screen
libGL error: failed to load driver: radeonsi
amdgpu: unknown (family_id, chip_external_rev): (145, 1)
libGL error: failed to create dri screen
libGL error: failed to load driver: radeonsi

**Comment by jose-rafael** [comment]
It happens to me too on 2 different systems, native repository works normally!

SO 1
OS: Arch Linux x86_64 
Kernel: 6.8.1-arch1-1 
DE: KDE Plasma 6.0.2
Flatpak: 1.15.6
MEGAsync: 5.2.0.0

SO 2
OS: Fedora Linux 40 x86_64 
Kernel: 6.8.1-300.fc40.x86_64
DE: KDE Plasma 6.0.2
Flatpak: 1.15.6
MEGAsync: 5.2.0.0

**Comment by nicolasdb** [comment]
Still the same problem here. 
I just did a fresh install of the last manjaro KDE, on the exact same laptop, then flatpak install mega
I was able to setup my account then I am only able to launch the app. Any attempt to access on the GUI and mega shutdown.
[24.05.05 flatpak mega log.txt](https://github.com/flathub/nz.mega.MEGAsync/files/15212610/24.05.05.flatpak.mega.log.txt)
. 

```
Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.11
KDE Frameworks Version: 5.115.0
Qt Version: 5.15.12
Kernel Version: 6.6.26-1-MANJARO (64-bit)
Graphics Platform: Wayland
Processors: 4 × Intel® Core™ i7-3520M CPU @ 2.90GHz
Memory: 11,5 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 4000
Manufacturer: LENOVO
Product Name: 2344CTO
System Version: ThinkPad T430
```

> same here on
> 
> ```
> Operating System: Manjaro Linux
> KDE Plasma Version: 5.26.5
> KDE Frameworks Version: 5.103.0
> Qt Version: 5.15.8
> Kernel Version: 6.1.12-1-MANJARO (64-bit)
> Graphics Platform: X11
> Processors: 4 × Intel® Core™ i7-3520M CPU @ 2.90GHz
> Memory: 11,5 GiB of RAM
> Graphics Processor: Mesa Intel® HD Graphics 4000
> Manufacturer: LENOVO
> Product Name: 2344CTO
> System Version: ThinkPad T430
> ```
> 
> I did a fresh install of manjaro on the 13 fev. 23 , then install mega with flatpack, it worked for a time, then I ran on this problem after an update. I'm not sure if it come from an update system or an update flatpak. It's still sync files ok but crash as soon as I click the icon.

**Comment by MagmaBro123** [comment]
Is this gonna be fixed?
